### DISCUSSION TITLE
相关文档   这个模块，可以配置成wiki样式么

### DISCUSSION THREAD

**Discussion by lujay** [hhyo/Archery#2139] [Q&A]
![image](https://user-images.githubusercontent.com/6263244/234744379-00b9bba4-4a27-4e6c-a6ae-e7719b048198.png)
版本 1.9.1 
我想把Wiki 拉下来直接放在  这个模块中， 但是这个模块目前好像只能放单个md文件，

**Comment by LeoQuote** [comment]
目前好像不支持多个文件，你可以在里面只写一个wiki的链接，欢迎pr
